rag/rag_practice/yes24_bestseller_crawling.py

```
@@ -0,0 +1,99 @@
+        print(f"[{index}] {title} | {author} | {price} | {tags_str}")
+
+        dic.append({
+            '제목': title,
+            '저자': author,
+            '가격': price,
+            '태그': tags_str,
+            'Link': book_url
+        })
+
+    except Exception as e:
+        continue
```

Ruff (0.13.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)